web-api/src/business/useCases/document/batchDownloadDocketEntriesInteractor.ts

```diff
@@ -34,23 +35,25 @@ export const batchDownloadDocketEntriesInteractor = async (
       authorizedUser,
     );
   } catch (error: any) {
-    const { userId } = authorizedUser;
     const { clientConnectionId, docketNumber } = downloadDocketEntryRequestInfo;
 
     const erMsg = error.message || 'unknown error';
     applicationContext.logger.error(
       `Error batch-downloading documents from case: ${docketNumber} - ${erMsg}`,
       { error },
     );
-    await applicationContext.getNotificationGateway().sendNotificationToUser({
-      applicationContext,
-      clientConnectionId,
-      message: {
-        action: 'batch_download_error',
-        error,
-      },
-      userId,
-    });
+    if (isAuthUser(authorizedUser)) {
```

Comment:
Tiny nitpick: would we rather check the authuser before calling `batchDownloadDocketEntriesHelper` than before sending the error message? Might be missing something